frontend/lib/map/map/map.dart

```diff
@@ -117,6 +117,9 @@ class _MapState extends State<Map> implements MapController {
 
   void _onMapCreated(MapboxMapController controller) {
     _controller = controller;
+    _controller.onSymbolTapped.add((argument) {
+      widget.onNoFeatureClick();
+    });
```

Comment:
It is deselecting a POI if you click it again. Otherwise there's just no feedback if you click on an icon multiple times.

Reply:
With this change it is no longer possible to:
* First select one poi
* Directly select another poi and change to it

Now it is like this:
* First select one poi
* When clicking the second poi the first one gets deselected and no new one gets selected

Reply:
Also, isn't the symbol handler also called if you directly click on the initial poi? I suspect that there could be a race condition between the symbol and the onmapclick handler.

I think with a single tap both can be called or am I wrong?

Reply:
A `Symbol` in these flutter bindings is (at least on Android) only the manually added symbols/icons (i.e. the overlaying icon of a selected store). Hence the symbol handler is only called for selected stores.
Also at least on Android, clicking on the Symbol (i.e. the selected store) does not call the .onmapclick handler.

On Android it does the expected thing:
* Click on the first poi => poi gets selected
* Click on a completely different poi => new poi gets selected.

Reply:
Oke then I'd say we move this discussion to a separate issue because it behaves differently on iOS/Android: https://github.com/ehrenamtskarte/ehrenamtskarte/issues/381